src/js/components/Filter/FilterForm.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import { Form } from 'react-final-form';
+
+import FilterVisibilityToggler from 'components/Filter/FilterVisibilityToggler';
+import ButtonPrimary from 'components/form-elements/ButtonPrimary';
+import ButtonTransparent from 'components/form-elements/ButtonTransparent';
+import TextField from 'components/form-elements/TextField';
+import { renderFormField } from 'utils/form-utils';
+import Translate from 'utils/Translate';
+
+import 'components/Filter/FilterStyles.scss';
+
+
+const FilterForm = ({
+  filterFields, onSubmit, searchFieldPlaceholder, formProps,
+}) => {
+  // Replace with SearchField created in another ticket
+  const searchField = {
+    type: TextField,
+    attributes: {
+      placeholder: searchFieldPlaceholder,
+    },
+  };
+  // Create initialValues from filterFields as empty values
+  const initialValues = Object.keys(filterFields).reduce((acc, curr) => {
+    if (!acc[curr]) {
+      return {
+        ...acc,
+        [curr]: '',
+      };
+    }
+    return acc;
+  }, { name: '' });
+  const [amountFilled, setAmountFilled] = useState(0);
+  const countFilled = (values) => {
+    // Calculate which object's values are not empty
+    setAmountFilled(Object.keys(values).filter(key => values[key]).length);
```

Comment:
A better solution would probably be to do `Object.values`
`Object.values(values).filter((val) => val).length`

Reply:
but also remember about `falsy` values like `0`.
If value of a filter will be `0` it will result as `false` during the filter and will not be counted

Reply:
that's nice point, I probably didn't consider this as I was only using string-like fields for testing. Thanks for that